vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2799,20 +2802,11 @@ def create_attn_groups(
 
         for kv_cache_group_spec in kv_cache_config.kv_cache_groups:
             kv_cache_spec = kv_cache_group_spec.kv_cache_spec
-            if isinstance(kv_cache_spec, AttentionSpec):
-                attn_backends = get_attn_backends_for_layers(
-                    kv_cache_group_spec.layer_names)
-            # TODO(lucas): move `get_mamba_attn_backend` into the mamba
-            # layers like above
-            elif isinstance(kv_cache_spec, MambaSpec):
-                attn_backends = {
-                    get_mamba_attn_backend(kv_cache_spec.mamba_type):
-                    kv_cache_group_spec.layer_names
-                }
-            else:
+            if not isinstance(kv_cache_spec, (AttentionSpec, MambaSpec)):
```

Comment:
Feel that we don't need this check now. We can assume all layers have implemented `get_attn_backend`.

Reply:
Thanks for clarifying this, Done!